gamemodes/terrortown/gamemode/shared/lang/en.lua

```diff
@@ -1237,17 +1237,17 @@ L.item_no_fall_damage_desc = [[Makes you immune to fall damage.]]
 L.item_no_fire_damage = "No Fire Damage"
 L.item_no_fire_damage_desc = [[Makes you immune to fire damage.]]
 L.item_no_hazard_damage = "No Hazard Damage"
-L.item_no_hazard_damage_desc = [[Makes you immune to hazard damage such as posion, radiation and acid.]]
+L.item_no_hazard_damage_desc = [[Makes you immune to hazard damage such as poison, radiation, and acid.]]
 L.item_no_energy_damage = "No Energy Damage"
-L.item_no_energy_damage_desc = [[Makes you immune to energy damage such as lasers, plasma and lightning.]]
+L.item_no_energy_damage_desc = [[Makes you immune to energy damage such as lasers, plasma, and lightning.]]
```

Comment:
While I do like the oxford comma i don't think we use it anywhere else. We should decide if we want to use it or not and use it consistently (or don't use it respectively)

Reply:
I think it should be used since it was used almost in every project where I was translating something. I have also seen that many study books and mass media use it, but not every, of course.

Reply:
Tbh, most of all I'm against the oxford style. It feels unnutural, most of all caused by the short grammatical constructs, which could easily be avoided and so would go hand in hand with many of other languages. Additionally, oxford style theoretically is just needed if the context is unclear. But "such as" already implies a list or single item, but not a deeper desccription with another list of an example.

Reply:
I don't split writing styles on Oxford and non-Oxford. In the Russian language, we have consistent and clear grammar rules, while in the English language, some things are unclear. I was taught to use this comma in such places, I see that comma many times, and that's why I want to put it there. If the TTT2 team doesn't want a comma in such places, I will not understand that decision, but I will accept it.

Reply:
I don't really favor any of them. I'm fine with both.

Reply:
after thinking about it again, i'm inclined to agree with alf here. in this case the oxford comma doesn't serve a purpose.
furthermore i'm afraid that it might confuse future translators who aren't familiar with it. The English language file is our 'reference' language afterall and therefore should be as simple as possible and only as complex as needed

Reply:
> furthermore i'm afraid that it might confuse future translators who aren't familiar with it. The English language file <…> should be as simple as possible and only as complex as needed

For me, it sounds like someone just said that you can translate without enough knowledge... heh, translation work isn't easy. Also, there can be more confusing things than just a little comma. (especially in the Russian language ;) )

I have to say that this comma doesn't really matter for my language since we have different grammar rules, so we don't put it there (_I bet that for other languages it doesn't matter too._). It only matters for players who use English. This comma is usually used in American English. For example, Steam and Valve games use it, while Garry's Mod... well, there are not enough in-game strings to understand if it uses it or not, but GMod's blogs use it too. So, I don't see what can cause confusion here for translators, except for the lack of knowledge.

Reply:
@TimGoll @lebroomer Any additional personal oppinion?

Reply:
We decided to not use the oxford comma in the translations.

Reply:
The comma is now removed.